apps/condo/domains/ticket/hooks/useTicketTableFilters.tsx

```diff
@@ -91,6 +96,11 @@ export function useTicketTableFilters (): Array<FiltersMeta<MeterReadingWhereInp
         { label: PaidMessage, value: 'isPaid' },
         { label: EmergencyMessage, value: 'isEmergency' },
         { label: WarrantyMessage, value: 'isWarranty' },
+        { label: ReturnedMessage.toLowerCase(), value: 'statusReopenedCounter' },
+    ]
+    const reviewValueOptions = [
+        { label: GoodReviewMessage, value: '2' },
```

Comment:
May value of review be stored as a more readable datatype (like `enum`)

Reply:
used enums 'returned, good, bad'